transport/transport.go

```diff
@@ -506,3 +507,50 @@ func wait(ctx context.Context, closing <-chan struct{}, proceed <-chan int) (int
 		return i, nil
 	}
 }
+
+const (
+	spaceByte   = byte(int(' '))
```

Comment:
what is with all the conversions? neither are needed.

See https://blog.golang.org/constants